src/SprykerEco/Zed/UnzerApi/Business/Api/Request/Converter/AuthorizeRequestConverter.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+/**
+ * MIT License
+ * For full license information, please view the LICENSE file that was distributed with this source code.
+ */
+
+namespace SprykerEco\Zed\UnzerApi\Business\Api\Request\Converter;
+
+use Generated\Shared\Transfer\UnzerApiRequestTransfer;
+use SprykerEco\Zed\UnzerApi\Business\Api\Request\UnzerApiRequestConstants;
+
+class AuthorizeRequestConverter implements UnzerApiRequestConverterInterface
+{
+    /**
+     * @param \Generated\Shared\Transfer\UnzerApiRequestTransfer $unzerApiRequestTransfer
+     *
+     * @return array{amount: float|null, currency: null|string, resources: array{customerId: null|string, typeId: null|string}, returnUrl: null|string}
```

Comment:
Please, don't use array shapes  for now, regarding the [RFC](https://spryker.atlassian.net/wiki/spaces/RFC/pages/2772435031/INTEGRATED+RFC+Array+docblock+syntax+update) it  can be available just for phpstan-* specific annotations.
Fix it in other places too.

Reply:
Fixed